README.md

```diff
@@ -364,6 +364,27 @@ In addition to columns, `sort` can also sort an arbitrary expressions.
 * | json | sort by length(endpoint_url)
 ```
 
+##### Timeslice
+`timeslice(<timestamp>) <duration> [as <field>]`: Truncates a timestamp to the
```

Comment:
It looks like duration is optional but defaults to 1h. I think it's probably better to just error during typecheck if a duration isn't set and remove the default.

Reply:
changed to raise an error